### ISSUE TITLE
Firefox 55.0.3 (64-bit) extension is outdated

### ISSUE THREAD

**Issue by qianyan** [adam-p/markdown-here#454]
![Markdown Here 2.13.2 Legacy](https://user-images.githubusercontent.com/6028514/30771104-594b370e-a005-11e7-959c-31fff5e05d35.png)

**Comment by antonio-gil** [comment]
Hi!

I guess that you still have the old version installed (the XUL one). In my case I've the new version and it looks like [this](https://www.dropbox.com/s/mddmod7y6fleu5r/2017-09-24_20h37_08.png?dl=0) (sorry, for an unknown reason I can't upload the image to Github).

I've installed the version number 2.13.1, and also, in the [developer page](https://addons.mozilla.org/en-US/firefox/user/adam-p/) of @adam-p on Firefox Addons page, you can see both extensions listed there.

I hope this helps.

Best regards,.

**Comment by qianyan** [comment]
cool, 2.13.1 works for me, but 2.13.2 is still marked legacy.

**Comment by antonio-gil** [comment]
Do you have both extensions installed?

2.13.2, as far as I can see, is marked as `Legacy` because is the "_old_" extension (I mean, the XUL/Classic one), while 2.13.1 is the "_newest_" one, or the one which fulfill the new requirements established for Mozilla regarding to extensions.

Or, well, TBHG, that's what I understand.

**Comment by qianyan** [comment]
Yes, I have installed 2.13.2 before, which was marked as `Legacy`. That is why I raised this issue. BUT when I followed what you comment above, installing 2.13.1, everything worked well.

Intuitively, 2.13.2 must be newer than 2.13.1 and also should be compatible with 2.13.1. That is the only way I have a little  concern.

BTW, what you understand is right.
